src/Gui/Quarter/Quarter.cpp

```diff
@@ -121,22 +121,23 @@
   \subpage examples
 */
 
-/*!
-  \page examples More Examples
-
-  The examples code is included in Quarter and can be found in the
-  src/examples subdirectory.
-
-  \subpage directui
-
-  \subpage dynamicui
-
-  \subpage inheritui
-
-  \subpage mdi
-
-  \subpage examiner
-*/
+// These pages are not exitant in this copy of Quarter
```

Comment:
Well I am open to suggestions.
I think the needed part of a version of Quarter was transferred into the FreeCad source tree but not the examples which is referred to in the doxygen comments. But I didn't want to remove the original text. The added comment could be somewhat more explanatory. It was a quick shot when I tried to make the SourceDocu run "properly" (i.e. better than before, still some errors) in order to actually get to the documentation